sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientBuilderUnitTest.java

```diff
@@ -47,33 +47,11 @@ public void invalidProtocol() {
         });
     }
 
-    /**
-     * Test for null API key
-     */
-    @Test
```

Comment:
Why are the tests removed?

Reply:
Now that we have two overloads for the `credential` in the builder, it cannot resolve at compile time to pass in null API key.

Reply:
assigning the null value to a type value, such as AzureKeyCredential credential = null, and use replace null with credential will resolve it.

Reply:
You can also do cast in API.
`.credential((AzureKeyCredential) null)`